packages/grid/x-data-grid-premium/src/components/GridColumnMenuAggregationItem.tsx

```diff
@@ -1,13 +1,14 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { GridColumnMenuItemProps, useGridSelector } from '@mui/x-data-grid-pro';
-import MenuItem from '@mui/material/MenuItem';
+import MUIMenuItem from '@mui/material/MenuItem';
 import ListItemIcon from '@mui/material/ListItemIcon';
 import ListItemText from '@mui/material/ListItemText';
 import FormControl from '@mui/material/FormControl';
 import InputLabel from '@mui/material/InputLabel';
 import { unstable_useId as useId } from '@mui/utils';
-import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { SelectChangeEvent } from '@mui/material/Select';
```

Comment:
Maybe replace `SelectChangeEvent` with `{ target: { value: string } }` to avoid imports from `@mui/material/Select`?

Reply:
Why I kept it here is, I thought most transpilers strip off type imports on production builds, due to which it should be OK to use types from these packages, am I missing something here?

Reply:
I think that the point Andrew raised is that this import will fail once we make @mui/material an optional peer dependency.

Reply:
Ah I see